cmd/autofix_test.go

```diff
@@ -1,13 +1,49 @@
 package cmd
 
 import (
+	"bufio"
+	"bytes"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-
-	"github.com/go-test/deep"
 )
 
+func compareFiles(file1, file2 string) bool {
+	f1, err := os.Open(file1)
+	if err != nil {
+		return false
+	}
+
+	f2, err := os.Open(file2)
+	if err != nil {
+		return false
+	}
+
+	s1 := bufio.NewScanner(f1)
+	s2 := bufio.NewScanner(f2)
+
+	for s1.Scan() {
+		s2.Scan()
+		if !bytes.Equal(s1.Bytes(), s2.Bytes()) {
```

Comment:
I'm sure this is more efficient but would it be more convenient for debugging to compare strings so that if the test fails it is easy to see the difference?

Reply:
Changed it to `.Text()` and added a `printf` for it so that you get notified about the difference.